default/generated/spring-framework/spring-framework-5.x-to-6.0-web-applications.yaml

```diff
@@ -49,3 +49,21 @@
     url: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-6.0-Release-Notes#web-applications
   - title: 'Specific Spring Framwork change'
     url: https://github.com/spring-projects/spring-framework/issues/28552
+
+- ruleID: spring-framework-5.x-to-6.0-web-applications-00002
+  category: mandatory
+  effort: 1
+  labels:
+  - konveyor.io/source=spring5
+  - konveyor.io/target=spring6+
+  when:
+    java.referenced:
+      pattern: org.springframework.http.HttpMethod
```

Comment:
Here we have two options:
a. Leave it as it is, and change the category to "potential". We would get many false positives.
b. Switch it to something like:
```yaml
    java.referenced:
      pattern: EnumSet<HttpMethod>
      location: TYPE
```
This wouldn't catch usages of switches, but we would get less false positives. What do you think @rromannissen ?

Reply:
From what I'm thinking, this is a pretty minor change when it comes to individual instances of using the ENUM.  So calling it out once for the whole CLASS as a "1" effort seems more appropriate than calling out each instance.